users/urls.py

```diff
@@ -1,7 +1,8 @@
 from django.urls import path
-from .views import SignUpView, SignInView
+from .views import SignUpView, SignInView , UserView
 
 urlpatterns = [
     path('/signup', SignUpView.as_view()),
     path('/signin', SignInView.as_view()),
+    path('/mypage', UserView.as_view()),
```

Comment:
```suggestion
    path('/me', UserView.as_view()),
```
해당 페이지를 참고해 봤을 때 다음과 같이 바꾸는 것이 좋을 것 같습니다.
https://stackoverflow.com/questions/36520372/designing-uri-for-current-logged-in-user-in-rest-applications

Reply:
수정했습니다!!!